knimin/tests/test_ag_new_barcode.py

```diff
@@ -0,0 +1,131 @@
+from unittest import main
+
+from tornado.escape import url_escape
+
+from knimin.tests.tornado_test_base import TestHandlerBase
+from knimin import db
+
+
+class TestAGBarcodePrintoutHandler(TestHandlerBase):
+    def test_get_not_authed(self):
+        response = self.get('/ag_new_barcode/download/')
+        self.assertEqual(response.code, 405)  # Method Not Allowed
+
+    def test_post(self):
+        self.mock_login_admin()
+        response = self.post('/ag_new_barcode/download/',
+                             {'barcodes': "1111,222,33,4"})
+        self.assertEqual(response.code, 200)
+        # is it sufficient to check that the response provides a pdf-file or
+        # should I somehow check the contents of the PDF?
```

Comment:
agree, on the assumption the contained functionality is reasonably tested. checking the pdf would be a pain and i dont think worth the effort